api-client-util/src/main/resources/infra/categories.json

```diff
@@ -940,6 +951,81 @@
 				"sunw"
 			],
 			"labels": ["Java-lang"]
+		},
+
+		{
```

Comment:
We should gather all our tiers under a .Net section (Could be just a comment prefixes all of this). Then, we should be sorted alphabetically (yes, I know java isn't)